Application/Ed-Fi-Ods.sln

```diff
@@ -285,16 +272,10 @@ Global
 		{DFD89624-8424-4878-8DF4-3C7C449AF990}.Npgsql|Any CPU.Build.0 = Debug|Any CPU
 		{DFD89624-8424-4878-8DF4-3C7C449AF990}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{DFD89624-8424-4878-8DF4-3C7C449AF990}.Release|Any CPU.Build.0 = Release|Any CPU
-		{197952F6-BFC3-481D-A94B-1DA0CF5FDA01}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{197952F6-BFC3-481D-A94B-1DA0CF5FDA01}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{197952F6-BFC3-481D-A94B-1DA0CF5FDA01}.Npgsql|Any CPU.ActiveCfg = Debug|Any CPU
-		{197952F6-BFC3-481D-A94B-1DA0CF5FDA01}.Npgsql|Any CPU.Build.0 = Debug|Any CPU
-		{197952F6-BFC3-481D-A94B-1DA0CF5FDA01}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{197952F6-BFC3-481D-A94B-1DA0CF5FDA01}.Release|Any CPU.Build.0 = Release|Any CPU
 		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.ActiveCfg = Npgsql|Any CPU
-		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.Build.0 = Npgsql|Any CPU
+		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
We're being inconsistent with Npgsql configs, some are using Debug|Any CPU and some are using Release|Any CPU.  Should they all be Debug?